webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9732,7 +9732,7 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                         }
                         if (kommunicateCommons.isObject(resp.message) && resp.message.groupId && resp.message.groupId == tabId && resp.message.metadata) {
                             CURRENT_GROUP_DATA.tabId = resp.message.groupId;
-                            if (resp.message.metadata.KM_STATUS === KommunicateConstants.CONVERSATION_CLOSED_STATUS) {
+                            if (resp.message.metadata.KM_STATUS === KommunicateConstants.CONVERSATION_CLOSED_STATUS || resp.message.metadata.KM_STATUS === KommunicateConstants.CONVERSATION_RESOLVED_STATUS) {
```

Comment:
safe side, use .toLowerCase() for comparison, that will cover old and new ones all status in case of case changes.